### PR TITLE
Making rigs have slowdown by default, and ERT rigs. Plus buffing HAPT armor rig.

### PR DESCRIPTION
**Pull request by Sindorman**
This PR has a refactor and a balance.

- All rigs now have slowdown of 1 by default, it was refactored that definitions derived from base rig no longer has it overridden as they used 1 anyway.

- ERT rig suits now have slowdown of 1. Implements #6027 

- After discussing with Arrow we decided that HAPT rig has to have higher values, since it is admibus. And also remove it from chance of spawning in rare loot.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
???

**Comment by Alberyk** [comment]
Light hardsuits, such as the stealh and ninja ones should not have a slowdown.

**Comment by Sindorman** [comment]
@Alberyk They do not have slowdown, most of rigs override base definition. This PR just transfers any that have `slowdown = 1` to the parent definition

File: html/changelogs/Sindorman-rig.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - tweak: "Rigs by default now have slowdown. Does not affect any rigs, only ERT."
```
Comment (ParadoxSpace) [inline html/changelogs/Sindorman-rig.yml]: Extraneous changelog entry.

**Pull request merged**